### PR TITLE
Add context menu to ListPanel

### LINKED ISSUE #883: Right click issues to show possible actions
For example, to assign labels like the 'L' shortcut key

### PR DESCRIPTION
**Pull request by ndt93**
Fixes #883

image

image

### CODE REVIEW

**Comment by damithc** [comment]
For me the most important role of this context menu is that it can help users to discover the keyboard shortcut. So I would like the menu to mention the shortcut key.

**Comment by ndt93** [comment]
All right, I will add that in.

**Comment by ndt93** [comment]
There is a problem though. If the shortcuts are added as accelerator to the context menu, they won't be reached through our manual keyboard shortcuts handler. If that's not desirable then the shortcuts can be added just as a text of the context menu.

**Comment by damithc** [comment]
I meant just as text, as a form of documentation.

**Comment by ndt93** [comment]
Seems like there are problems with testing the context menu on CI. Will take some times to fix.

**Comment by ndt93** [comment]
@lejolly Can help review this? Thanks.

File: src/main/java/ui/listpanel/ListPanel.java [hunk 1]
```diff
@@ -263,6 +258,73 @@ private void setupKeyboardShortcuts() {
...
+
+    private MenuItem updateMarkAsReadUnreadMenuItem() {
+        Optional<TurboIssue> item = listView.getSelectedItem();
+        if (item.isPresent()) {
+            markAsReadUnreadMenuItem.setDisable(false);
+            TurboIssue selectedIssue = item.get();
+
+            if (selectedIssue.isCurrentlyRead()) {
+                markAsReadUnreadMenuItem.setText(markAsUnreadMenuItemText);
+
```
Comment (lejolly) [inline src/main/java/ui/listpanel/ListPanel.java]: extra line?

Comment (ndt93) [inline src/main/java/ui/listpanel/ListPanel.java]: all right, line removed.

**Comment by lejolly** [comment]
looks good, go ahead and merge it.

**Pull request merged**